assets/js/SetExpiration.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import { signConfigType } from './types';
+
+function stringify(expires) {
+  if (expires) {
+    return expires.toISOString();
+  }
+
+  return null;
+}
+
+function updateConfig(setConfigsFn, realtimeId, signConfig, expires) {
+  const expirationConfig = stringify(expires);
+
+  const newConfig = {
+    ...signConfig,
+    expires: expirationConfig,
+  };
+
+  setConfigsFn({
+    [realtimeId]: newConfig,
+  });
+}
+
+function parseDate(str) {
+  if (str) {
+    return new Date(str);
```

Comment:
Do we want to handle the case where the date string is invalid? It would require something to go wrong in another part of the application so I'm not sure it's worth it.